### PR TITLE
Fix danger zone text taking full width in medium and large screen sizes

### PR DESCRIPTION
**Pull request by pavinduLakshan**
### Purpose

$subject
https://github.com/wso2/identity-apps/assets/41533942/c4ea2102-a852-40d5-9d70-c6af4062d60c

### Related Issues
- https://github.com/wso2/product-is/issues/17171

### Related PRs
- N/A

### Checklist
- [ ] e2e cypress tests locally verified.
- [x] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [x] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [x] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: modules/theme/src/themes/default/elements/segment.overrides [hunk 1]
```diff
@@ -83,12 +83,10 @@
...
 
-        @media only screen and (max-width: 1512px) {
-            .header {
-                max-width: 75%;
-            }
+        .header-wrapper {
+            max-width: 75%;
         }
-
+
```
Comment (Achintha444) [inline modules/theme/src/themes/default/elements/segment.overrides:89]:
```suggestion
```

File: .changeset/five-mayflies-compare.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+"@wso2is/react-components": patch
+"@wso2is/theme": patch
```
Comment (Achintha444) [inline .changeset/five-mayflies-compare.md:3]: I think we need to bump, console and myaccount versions as well ? Right @JayaShakthi97

Comment (pavinduLakshan) [reply .changeset/five-mayflies-compare.md]: Fixed with 2fb7b58

File: modules/theme/src/themes/default/elements/segment.overrides (hunk 1 above)
Comment (pavinduLakshan) [reply modules/theme/src/themes/default/elements/segment.overrides]: Fixed with 3b9eafd

**Review by Achintha444 — APPROVED** [review]

**Pull request merged**